File: app/src/main/java/com/jann_luellmann/thekenapp/view/CustomerRow.java
package com.jann_luellmann.thekenapp.view;

import android.content.Context;
import android.widget.TableRow;
import android.widget.TextView;

import com.jann_luellmann.thekenapp.R;

public class CustomerRow extends TableRow {

    private TextView textview;

    public CustomerRow(Context context) {
        super(context);
        init();
    }

    private void init() {
        inflate(getContext(), R.layout.tablerow, this);
        textview = findViewById(R.id.customer);
    }

    public void setText(String text) {
        textview.setText(text);
    }
}

File: app/src/main/java/com/jann_luellmann/thekenapp/view/CustomerRow.kt
package com.jann_luellmann.thekenapp.view

import android.content.Context
import android.widget.TextView
import com.jann_luellmann.thekenapp.R
import android.widget.TableRow

class CustomerRow(context: Context?) : TableRow(context) {
    private var textview: TextView? = null
    private fun init() {
        inflate(context, R.layout.tablerow, this)
        textview = findViewById(R.id.customer)
    }

    fun setText(text: String?) {
        textview!!.text = text
    }

    init {
        init()
    }
}